src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -57,10 +57,12 @@
   private final String START_SUBSTRING = "| ";
   private final String END_SUBSTRING = "followers";
   private final int ALLOWED_NUMBER_OF_MATCHING_BUSINESSES = 5;
+  private final String BIG_BUSINESSES_DATABASE = "BigBusinesses";
+  private final String SMALL_BUSINESSES_DATABASE = "SmallBusinesses";
   private LatLng latLng;
   private List<Listing> allBusinesses;
   private int numberOfSmallBusinesses;
-  
+  private boolean isBigBusiness;
```

Comment:
As written, this says your BusinessesService is a big business (or is not, depending on the value of this boolean). This is an anti-pattern in object oriented programming. A better design here would be to use a function. A function that take a Listing object, executes the three checks that are being used to determine whether or not a business is big, and returns true/false based on that. You can then use that on line 163 in the same way that you're using the boolean.

Reply:
I changed the flow of the program so that the functions return if the business is big or not. The only problem I am facing is that since you can not return something within an if statement without returning something at the end of the function. I had to make multiple boolean variables within each function to either return true or false at the end of each appropriate function.

Reply:
I'm not seeing any updates in the code. Can you try re-pushing your commit?

Reply:
The code is up to date, look at the commit under this comment. (changed function to return true or false if the business is big)